utils/storybook/customThemes.ts

```diff
@@ -0,0 +1,22 @@
+export const customColorTheme = {
```

Comment:
Perhaps we want this to be a partial of the `CanvasTheme` object? This way we could get autocomplete.

Reply:
We have a `PartialCanvasTheme` type 😉 . `Partial<>` doesn't work because it's not recursive.